### PR TITLE
FEATURE: WEB-1683 Redesign GoodData.UI homepage

### PR DESCRIPTION
**Pull request by czechian**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/508/ : SUCCESS in 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/297/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/509/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/298/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/515/ : SUCCESS in 57s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/302/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/516/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/303/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/517/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/304/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/518/ : SUCCESS in 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/305/ : SUCCESS in 4s

File: website/static/js/parallax.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-        homeSplashBgOverflow = 30,
-        y, percentage;
+    var splash = document.getElementById('splash-parallax'),
+        layers = splash.childNodes,
+        height = 200,
+        scrollProgress;
 
-    window.addEventListener('scroll', function() {
-        y = window.scrollY;
+    layers.forEach(function(layer, index) {
```
Comment (kandl) [inline website/static/js/parallax.js:7]: IE11 does not support forEach on NodeList -> https://developer.mozilla.org/en-US/docs/Web/API/NodeList
Add polyfill, or convert it to an array first (eg `var layersArray = Array.prototype.slice.call(layers);`)

File: website/static/js/gaEvents.js [hunk 2]
```diff
@@ -1,31 +1,31 @@
 document.addEventListener('DOMContentLoaded', function() {
 
-	function sendGAEvent(item, action, category) {
```
Comment (kandl) [inline website/static/js/gaEvents.js:3]: Please avoid unnecessary formatting changes, it makes it hard to track changes. Or add prettier config...

File: website/static/js/parallax.js [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+        layer.style.transform = "translateY(" + scrollProgress() * index + "%)";
+    });
 
-        if(y <= homeSplashHeight) {
-            percentage = ((homeSplashBgOverflow / 2) * (-((homeSplashHeight - y) / homeSplashHeight))) * ((100 - homeSplashBgOverflow) / 100);
-            homeSplashBgElem.style.transform = 'translate3d(0, ' + percentage + '%, 0)';
-        }
+    window.addEventListener('scroll', function() {
+        layers.forEach(function(layer, index) {
```
Comment (kandl) [inline website/static/js/parallax.js:12]: It looks like a same function as few lines above.
Maybe extract it to something like:
```
function moveLayers(layers) {
  // Support IE11, it does not support forEach on NodeList https://developer.mozilla.org/en-US/docs/Web/API/NodeList
  var layersArray = Array.prototype.slice.call(layers);
  layersArray.forEach(function(layer, index){
    layer.style.transform = "translateY(" + (scrollProgress() * index) + "%)";
  })
}
```
and then `moveLayers(layers)`

File: website/static/js/toggleNav.js [hunk 4]
```diff
@@ -2,16 +2,18 @@ document.addEventListener('DOMContentLoaded', function() {
...
-		var NAV_OPEN = "toggleNavOpen";
+        var bodyClasses = document.body.classList;
+        var NAV_OPEN = "toggleNavOpen";
 
-    	navToggler.addEventListener('click', function() {
-    		if(bodyClasses.contains(NAV_OPEN)) {
-    			bodyClasses.remove(NAV_OPEN);
-    		}
-    		else {
-    			bodyClasses.add(NAV_OPEN);
```
Comment (kandl) [inline website/static/js/toggleNav.js:13]: Another unnecessary reformatting

Comment (czechian) [reply website/static/js/toggleNav.js]: Rolled back

File: website/static/js/gaEvents.js (hunk 2 above)
Comment (czechian) [reply website/static/js/gaEvents.js:3]: Rolled back

File: website/static/js/parallax.js (hunk 1 above)
Comment (czechian) [reply website/static/js/parallax.js]: Fixed

File: website/static/js/parallax.js (hunk 3 above)
Comment (czechian) [reply website/static/js/parallax.js:15]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/520/ : SUCCESS in 1m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/306/ : SUCCESS in 5s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/521/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/55/ : FAILURE in 2m 43s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/148/ : SUCCESS in 21s
